IndustrialUnitDatabase/SQLiteDataAccess.cs

```
@@ -0,0 +1,141 @@
+﻿using Dapper;
+using IndustrialUnit.Model;
+using Microsoft.Data.SqlClient;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SQLite;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace IndustrialUnitDatabase
+{
+  public static class SQLiteDataAccess
+  {
+    private static readonly string loadConnectionString = $"Data Source={Helper.DatabasePath("IndustrialUnitDB.db")}";
+
+    public static DataTable GetAll(string tableName)
+    {
+      if (File.Exists(Helper.DatabasePath("IndustrialUnitDB.db")))
+      {
+        using (var con = new SQLiteConnection(loadConnectionString))
+        {
+          con.Open();
+
+          string stm = $"SELECT * FROM {tableName}";
+          var cmd = new SQLiteCommand(stm, con);
+          SQLiteDataAdapter sda = new SQLiteDataAdapter(cmd);
+          DataTable dt = new DataTable(tableName);
+          sda.Fill(dt);
+          return dt;
+        }
+      }
+      else
+      {
+        throw new FileNotFoundException("Database not found!");
```

Fail fast. Általános szabály, hogy ha egy `else` blokkban csak egy `throw` van, akkor megfordítod a feltételt, így nincs felesleges nesting (lásd a másik kommentemben az újraírt változatot).